Sample.java

```diff
@@ -1,7 +1,132 @@
-// Time Complexity :
-// Space Complexity :
-// Did this code successfully run on Leetcode :
+// Time Complexity : O(n)
+// Space Complexity : O(n)
+// Did this code successfully run on Leetcode : Yes
 // Any problem you faced while coding this :
 
 
 // Your code here along with comments explaining your approach
+
+class Point{
+    int key,value;
+    Point(int key,int value){
+        this.key=key;
+        this.value=value;
+    }
+}
+
+class MyHashMap {
+    
+    List<List<Point>> list;
+    /** Initialize your data structure here. */
+    public MyHashMap() {
+        list = new LinkedList<>(); 
+        for(int index=0;index<10000;index++)
+            list.add(new LinkedList<Point>());
+    }
+    
+    public int exists(int key,int index) {
+        
+        List<Point> tmpList = list.get(index);
+        int pos = 0;
+        for(Point p:tmpList){
+    
+            if(p.key==key)
+                return pos;
+            pos++;
+        }
+        
+        return -1;
+    }
+    
+    /** value will always be non-negative. */
+    public void put(int key, int value) {
+        int newKey=key%10000;
+        int pos = exists(key,newKey);
+        
+        if(pos!=-1){
+            list.get(newKey).set(pos, new Point(key,value));
+        }else{
+            list.get(newKey).add(new Point(key,value));
+        }
+        
+    }
+    
+    /** Returns the value to which the specified key is mapped, or -1 if this map contains no mapping for the key */
+    public int get(int key) {
+        int newKey=key%10000;
+        int pos = exists(key,newKey); 
+        if(pos!=-1)
+            return list.get(newKey).get(pos).value;
+        
+        return -1;
+    }
+    
+    /** Removes the mapping of the specified value key if this map contains a mapping for the key */
+    public void remove(int key) {
+       int newKey=key%10000;
+        int pos = exists(key,newKey); 
+        // Point p = list.get(newKey).get(pos);
+            
+        if(pos!=-1)
+            list.get(newKey).remove(pos);
+    }
+}
+
+
+/**
+ * Your MyHashMap object will be instantiated and called as such:
+ * MyHashMap obj = new MyHashMap();
+ * obj.put(key,value);
+ * int param_2 = obj.get(key);
+ * obj.remove(key);
+ */
+
+// Time Complexity : Each of the operation will be O(1)
+// Space Complexity : Each of the operation will be O(1)
```

Comment:
Since we use a stack data structure, the time complexity will be O(n).

Reply:
Thanks @iivaishnavi for letting em know. You mean O(n) for all the operations or for single operation.